framework/source/class/qx/test/util/DateFormat.js

```diff
@@ -849,6 +858,10 @@ qx.Class.define("qx.test.util.DateFormat",
     dfDE.dispose();
     dfUS.dispose();
 
+  },
+  
+  hasNoEdge: function() {
+    return !(qx.core.Environment.get("browser.name") == "edge");
```

Comment:
Any reason not to do `return (qx.core.Environment.get("browser.name") != "edge");` instead? The `!=` operator is easier to read than a negated expression.

Reply:
same here. adopted from https://github.com/qooxdoo/qooxdoo/blob/master/framework/source/class/qx/test/io/request/XhrWithRemote.js#L227-L229